drivers/include/sen5x.h

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (C) 2023 TU Braunschweig Institut für Betriebssysteme und Rechnerverbund
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_sen5x Sensirion Embedded I2C SEN5x Driver
+ * @ingroup     drivers_sensors
+ * @ingroup     drivers_saul
+ * @brief       Driver for I2C communication to SEN5x devices.
+ *
+ * @{
+ *
+ * @file
+ *
+ * @author      Daniel Prigoshij <prigoshi@ibr.cs.tu-bs.de>
+ */
+
+#ifndef SEN5X_H
+#define SEN5X_H
+
+/* Add header includes here */
+
+#include "periph/i2c.h"
+#include <stdint.h>
+#include "saul.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Declare the API of the driver */
+
+/**
+ * @brief   Wrapper for measured values
+ */
+typedef struct {
+    uint16_t mass_concentration_pm1p0;      /** raw value is scaled with factor 10: PM1.0 [µg/m³] = value / 10 */
```

Comment:
```suggestion
    uint16_t mass_concentration_pm1p0;      /**< raw value is scaled with factor 10: PM1.0 [µg/m³] = value / 10 */
```

Normally the Doxygen comment applies tk what comes after the comment. Here it should apply to the code that is just before the comment. Using `/**<` instead of `/**` does this.

Reply:
Thanks, I've changed it now.